libzhlgen/test/functions/win32/1.6.9/ResourceControl.zhl

```diff
@@ -6,6 +6,8 @@ __thiscall GL_Primitive* ResourceControl::CreateImagePrimitive(ResourceControl *
 __thiscall void ResourceControl::OnInit(ResourceControl *this, int imageSwappingMode);
 "5589e557565389cf31db83e4f083ec408d442430c744240c10000000c744240800000000c744243064617461c74424342e646174894424288b4508":
 __thiscall char* ResourceControl::LoadFile(ResourceControl *this, const std::string& fileName);
+"5589e557565389ce31db83e4f083ec208b45088b7d0cc744240c10000000c7442408000000008b00894424048b81c8000000890424":
```

Comment:
Linux 64
`.4156b91000000041554531ed415455` (after `ResourceControl::LoadFile`)

Linux 32
`.5531ed57565383ec2c8b442444` (after `ResourceControl::LoadFile`)

Reply:
Just putting it here since I had this file up for something else in Ghidra, I think I'll still checkout your branch & add the signatures myself later

Reply:
I'll address the previous comment and rebase this PR today (approx 12 hrs later I guess). Feel free to checkout after that if you'd want to avoid merge conflicts.

Reply:
Were these signatures ever added? I don't see them.